src/foundry/client/data/documents/fog-exploration.d.mts

```diff
@@ -11,6 +13,23 @@ declare global {
    * The client-side FogExploration document which extends the common BaseFogExploration model.
    */
   class FogExploration extends ClientDocumentMixin(foundry.documents.BaseFogExploration) {
+    /**
+     * Obtain the fog of war exploration progress for a specific Scene and User.
+     * @param query      - Parameters for which FogExploration document is retrieved
+     * @param options    - Additional options passed to DatabaseBackend#get.
+     *                     (Default: `{}`)
```

Comment:
This is definitely a nitpick but we write `(default: ...)`, i.e. with a lowercase `d`. It probably doesn't matter that much but this convention has been around for a while.